RELEASE.md

```diff
@@ -127,7 +147,7 @@ git push origin "${tag}" "pkg/apis/monitoring/${tag}" "pkg/client/${tag}"
 
 Signed tag with a GPG key is appreciated, but in case you can't add a GPG key to your Github account using the following [procedure](https://docs.github.com/articles/generating-a-gpg-key), you can replace the `-s` flag by `-a` flag of the `git tag` command to only annotate the tag without signing.
 
-Our CI pipeline will automatically push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator)
+Once a tag is created, the release process through github actions will be triggered for this tag. Our CI pipeline will automatically push the container images to [quay.io](https://quay.io/organization/prometheus-operator) and [ghcr.io](https://github.com/prometheus-operator/prometheus-operator/pkgs/container/prometheus-operator). Wait till [publish](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/publish.yaml) and [release](https://github.com/prometheus-operator/prometheus-operator/actions/workflows/release.yaml) jobs are completed so that assets will be attached to release correctly.
 
 Go to https://github.com/prometheus-operator/prometheus-operator/releases/new, associate the new release with the before pushed tag, paste in changes made to `CHANGELOG.md` and click "Publish release".
```

Comment:
Do you want to add that we shouldn't use Draft mode? Or do you want to try fixing the action?